streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -255,9 +255,7 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
     }
   }
 
-  def numReceivers(): Int = {
-    receiverInputStreams.size
-  }
+  def numReceivers(): Int = receiverInputStreams.length
```

Comment:
This fine (I just would like to help for the next reviewer), so the reason of this change:

>Replace .size with .length on arrays and strings
>
>Inspection info: This inspection reports array.size and string.size calls. While such calls are legitimate, they require an additional implicit conversion to SeqLike to be made. A common use case would be calling length on arrays and strings may provide significant advantages.

Reply:
Yep, we ought to make this kind of change (and use `lengthCompare` on a similar note where sizes are compared) wherever perf matters. Probably a good habit in general, but, wouldn't change it unless the code is already changing. Here I just took the liberty of adjusting this one, though there's no big reason for it here.

More generally I have a number of large code cleanup changes I want to get into Spark 3, and although I won't change this particular issue wholesale, I do want to get in some code cleanup before a new major version.